libs/pipelines/api_pipeline.py

```diff
@@ -218,6 +217,34 @@ def build_county_summary_from_model_output(input_dir) -> List[APIOutput]:
     return results
 
 
+def remove_root(obj: dict):
+    """Removes __root__ and replaces with __root__ value.
+
+    When pydantic models with the key __root__ are used to represent
+    json lists are serialized using `pydantic_model.dict()` the "__root__"
+    key is included. This will remove that __root__ key and replace with a list.
+
+    A dictionary {"__root__": []} will return [].
+
+    Args:
+        obj: pydantic model as dict.
+
+    Returns: object with __root__ removed.
+    """
+    # Objects with __root__ should have it as the only key.
+    if len(obj) == 1 and '__root__' in obj:
+        return obj['__root__']
+
+    results = {}
+    for key, value in obj.items():
+        if isinstance(value, dict):
+            value = remove_root(value)
+
+        results[key] = value
```

Comment:
Do we really need to do this recursively?  Maybe I don't understand how we're using __root__.  I assumed it was only an issue for root arrays.

Reply:
we do because root can be on any `pydantic.BaseModel` object.

so if you had:
```
class FirstList(pydantic.BaseModel):
    __root__: List = pydantic.Field(...)


class SecondList(pydantic.BaseModel):
    __root__: List[FirstList] = pydantic.Field(...)

```

calling dict on an instance of `SecondList` would return:
```
{"__root__": [{"__root__": []}])
```
what we want is: `[[]]`

Reply:
I see. 👍 

Thanks for the explanation.